pages/login/login.js

```diff
@@ -0,0 +1,15 @@
+import Link from "next/link";
+
+export default function login() {
```

Comment:
Is this really supposed to be a login page? Or an overview page? Please use name of component and headings / page meaning consistently.

Reply:
it is a bit confusing now but the burgermenu icon should first lead to the login and then to the overview page

Reply:
Since you do not have a login, you should just do the Overview page now and add the login later! I recommend using a different Icon than the burgermenu to link to the overview page.